python/ray/tests/test_plasma_unlimited.py

```diff
@@ -16,6 +16,10 @@
 
 MB = 1024 * 1024
 
+# Note: Disk write speed can be as low as 6 MiB/s in AWS Mac instances, so we have to
+# increase the timeout.
+pytestmark = [pytest.mark.timeout(900 if platform.system() == "Darwin" else 180)]
```

Comment:
Should we update the bazel test timeout instead?

Reply:
@rynewang would increase bazel test timeout work here as well? If so I think that's better since we have a single place for controlling test timeout.